rtgui/adjuster.cc

```diff
@@ -317,32 +300,35 @@ void Adjuster::resetPressed (GdkEventButton* event)
     }
 }
 
-double Adjuster::shapeValue (double a)
+double Adjuster::shapeValue (double a) const
 {
-    double val = round(a * pow(double(10), digits)) / pow(double(10), digits);
+    const double pow10 = pow(10.0, digits);
+    const double val = round(a * pow10) / pow10;
     return val == -0.0 ? 0.0 : val;
 }
 
 void Adjuster::setLimits (double vmin, double vmax, double vstep, double vdefault)
 {
+    sliderChange.block(true);
+    spinChange.block(true);
 
-    sliderChange.block (true);
-    spinChange.block (true);
+    double pow10 = vstep;
+    for (digits = 0; fabs(pow10 - floor(pow10)) > 0.000000000001; digits++, pow10 *= 10.0);
```

Comment:
`std::fabs()` and friends? We include `<cmath>`, so everything should be prefixed with `std::`.

Reply:
Done